File: scoreTest.js
function scoreTest(str, right, omit, wrong){
    let corr = 0
    let wrong1 = 0
    let omit1 = 0 
    for (let x = 0; x < str.length; x++){
      if (str[x] == 0){
        corr++
      }else if (str[x] == 1){
        omit1++
      }else{
        wrong1++
      }
    }
    let totCor = corr * right
    let totOmit = omit1 * omit
    let totWrong = wrong1 * wrong
    let sum = totCor + totOmit - totWrong
    return sum
    
  }

File: scoreTest.py
def score_test(tests, right, omit, wrong):
    r1 = 0
    o1 = 0
    wr1 = 0
    for x in tests:
        if x == 0:
            r1 +=1
        elif x ==1:
            o1 += 1
        else:
            wr1 += 1
    r2 = r1* right
    o2 = o1 * omit
    wr2 = wr1 * wrong
    summ = r2 + o2 - wr2
    return summ